cmd/podman/containers/cp.go

```diff
@@ -43,7 +40,7 @@ var (
 func cpFlags(cmd *cobra.Command) {
 	flags := cmd.Flags()
 	flags.BoolVar(&cpOpts.Extract, "extract", false, "Extract the tar file into the destination directory.")
-	flags.BoolVar(&cpOpts.Pause, "pause", copyPause(), "Pause the container while copying")
```

Comment:
We should probably deprecate this - it's an unnecessary addition (we fixed the CVE it was supposed to resolve in a different way).

Reply:
Sounds good to me.  Maybe something for a follow up PR when working on `remote cp`.  Will spare us from adding the parameters.

I think we can leave it on the CLI but make it a NOP?  Or full removal given 3.0 would allow us to?

Reply:
Leave it but remove documentation, and hide the option from --help.